backend/sse-edge/src/main/kotlin/io/featurehub/edge/stats/StatsFeature.kt

```diff
@@ -12,8 +12,8 @@ import org.glassfish.jersey.server.spi.Container
 import org.glassfish.jersey.server.spi.ContainerLifecycleListener
 
 class StatsFeature : Feature {
-  @ConfigKey("edge.publish-stats-with-nats")
-  var publishUsingNats: Boolean? = true
+  @ConfigKey("edge.stats-publisher")
```

Comment:
this could be set to "pubsub" or "nats" is it?

Reply:
It defaults to nats now but I guess it can be anything depending on the config file